src/index.js

```diff
@@ -1 +1,5 @@
-'use strict';
+import netflixResult from 'netflix-result';
+
+export default function index(element) {
+
```

Comment:
Now that you have `imported` the `netflixResult` module use it and pass in the elements it requires: https://github.com/hannahgswift/10-netflix-result/pull/1/files#diff-e126d2a9a261c0521435bd55eac58caaR1